custom-ports/dawn/portfile.cmake

```diff
@@ -0,0 +1,126 @@
+# dawn
+vcpkg_from_git(
+   OUT_SOURCE_PATH SOURCE_PATH
+   URL https://dawn.googlesource.com/dawn
+   REF 615b5fd6606a4d2bed2dc13a95ea9f87b497dec4 # chromium/6740
+   HEAD_REF main
+)
+
+# DAWN_FETCH_DEPENDENCIES doesn't work on Windows, so we'll fetch the dependencies ourselves
+if(VCPKG_HOST_IS_WINDOWS)
+   vcpkg_find_acquire_program(GIT)
```

Comment:
Weird that git would possibly not exist on a windows build machine?

Reply:
yeah I think it's just no on the path? At least in this environment? I can't remember the exact details.